test/NuGet.Clients.Tests/NuGet.CommandLine.Test/PortReserver.cs

```diff
@@ -22,6 +23,7 @@ namespace NuGet.CommandLine.Test
     public class PortReserver : IDisposable
```

Comment:
Actually, there are significant API changes here. Can you file an issue to consolidate this code?